src/CMakeLists.txt

```diff
@@ -85,6 +85,14 @@ endif()
 
 if(STK_STACK_TRACE)
     add_definitions(-DSTK_STACK_TRACE)
-    add_backward(stk)
-    target_link_libraries(stk PUBLIC ${BACKWARD_LIBRARIES})
+
+	if(UNIX)
+		find_package(Backward)
+		if(Backward_FOUND)
```

Comment:
I think I have found an elegant solution. `backward-cpp` is [packaged](https://aur.archlinux.org/packages/backward-cpp/) and available as a system library, so instead of bundling it we can leave it as an optional dependency. It will be used only if CMake finds it in the system, otherwise it will default to our implementation.

Reply:
Great, this is reasonable solution. This way the windows user won't need to clone a repo they don't even use. 😄